daemons/schedulerd/sched_allocate.c

```diff
@@ -1225,16 +1213,10 @@ cleanup_orphans(resource_t * rsc, pe_working_set_t * data_set)
             && pe_get_failcount(node, rsc, NULL, pe_fc_effective, NULL,
                                 data_set)) {
 
-            char *key = generate_op_key(rsc->id, CRM_OP_CLEAR_FAILCOUNT, 0);
-            action_t *clear_op = custom_action(rsc, key, CRM_OP_CLEAR_FAILCOUNT,
-                                               node, FALSE, TRUE, data_set);
-
-            add_hash_param(clear_op->meta, XML_ATTR_TE_NOWAIT, XML_BOOLEAN_TRUE);
+            pe_action_t *clear_op = NULL;
 
-            pe_rsc_info(rsc,
-                        "Clearing failure of %s on %s because it is orphaned "
-                        CRM_XS " %s",
-                        rsc->id, node->details->uname, clear_op->uuid);
+            clear_op = pe__clear_failcount(rsc, node, "it is orphaned",
+                                           data_set);
```

Comment:
log going from info to notice in that case is probably intentional ...

Reply:
one could argue that clearing due to being orphaned is expected and thus should be info, but I like the consistency of logging all clear ops at the same level, and it's easier on the implementation :)